CondFormats/SiPixelTransient/src/SiPixelTemplate2D.cc

```diff
@@ -626,6 +626,12 @@ bool SiPixelTemplate2D::getid(int id) {
 bool SiPixelTemplate2D::interpolate(int id, float cotalpha, float cotbeta, float locBz, float locBx) {
   // Interpolate for a new set of track angles
 
+  //check for nan's
+  if (!edm::isFinite(cotalpha) || !edm::isFinite(cotbeta)) {

```

Comment:
I think you want to maintain in the clause `#ifndef SI_PIXEL_TEMPLATE_STANDALONE` here, right?